### ISSUE TITLE
Voiceover cannot navigate to the pop-up context menu on iOS

### ISSUE THREAD

**Issue by haven2world** [microsoft/roosterjs#2043]
**Describe the bug**
For accessibility users, when voiceover is on, they can double tap and hold to call out the context menu. Then when they swipe right, the focus should move to the context menu. But currently it doesn't work on RoosterJS demo, OWA-mini and OM.

**To Reproduce**
1. Open Compose Page and type some words on iOS
2. Turn on VoiceOVer
3. Double tap and hold for a while, then the context menu will pop up
4. Swipe right to focus on the menu item

**Expected behavior**
The menu item should be able to focus

**Screenshots**
Expected behavior(here is a contenteditable Div)

https://github.com/microsoft/roosterjs/assets/17299418/e1b71e35-d5b8-431c-9658-e5360a8bd85c

Roosterjs demo behavior

https://github.com/microsoft/roosterjs/assets/17299418/c3c819f1-2046-44e3-b052-2f1832ae3475

**Device Information**
 - OS: iOS
 - Browser WebKit/Safari
 - Version Latest

**Additional context**

**Comment by haven2world** [comment]
I'm not sure if we can do anything on roosterJS side. At least, I'd like to know why the focus system breaks.

**Comment by haven2world** [comment]
Tried more examples, I think this is likely to be a webkit issue. Will ask them instead.
